example/cds-hooks-api-guide/src/prefetch-context/prefetch-context.js

```diff
@@ -7,117 +7,118 @@ import {
-            url: "https://www.automatemedical.com/",
-          },
-          summary: `Current Address`,
-          indicator: "info",
-        }),
-        // Gender
-        new Card({
-          detail: `${data.patient.gender}`,
-          source: {
-            label: "Automate Medical, Inc.",
-            url: "https://www.automatemedical.com/",
-          },
-          summary: `Gender`,
-          indicator: "info",
-        }),
-        // Telecom (only pulls value of first element in array)
-        new Card({
-          detail: `${telecom[0].value}`,
-          source: {
-            label: "Automate Medical, Inc.",
-            url: "https://www.automatemedical.com/",
-          },
-          summary: `Contact`,
-          indicator: "info",
-        }),
-        // Information on the last encounter
-        new Card({
-          detail: `Last visit was on ${
-            encounters.pop().resource.period.start
-          }. There are ${encounters.length} encounter${
-            encounters.length <= 1 ? "" : "s"
-          } on record.`,
-          source: {
-            label: "Automate Medical, Inc.",
-            url: "https://www.automatemedical.com/",
-          },
-          summary: `Last visit`,
-          indicator: "info",
-        }),
-        // Seeing the last encounter information
-        new Card({
-          detail: `Make a new appointment? ${
-            newApp[0] === true ? "Yes" : "No"
-          }, last appointment was ${newApp[1]} day${
-            newApp[1] > 1 ? "s" : ""
-          } ago.`,
-          source: {
-            label: "Automate Medical, Inc.",
-            url: "https://www.automatemedical.com/",
-          },
-          summary: `Book new appointment`,
-          indicator: "info",
-        }),
-      ],
-    };
-  }
-);
+};
+
+const handler = async (request) => {
+  const data = request.prefetch;
+  const patientNames = processPatientNames(data.patient);
+  const addresses = processAddresses(data.patient);
+  const telecom = processTelecom(data.patient);
+  const encounters = processEncounters(data.encounter);
+  const newApp = newAppointment(data.encounter, 100);
+  return {
+    cards: [
+      // Name(s)
+      new Card({
+        detail: `This patient has ${patientNames.length} name${
+          patientNames.length <= 1 ? "" : "s"
+        } on record.`,
+        source: {
+          label: "Automate Medical, Inc.",
+          url: "https://www.automatemedical.com/",
+        },
+        summary: `Now seeing: ${patientNames[0].given} ${patientNames[0].family}.`,
+        indicator: "info",
+      }),
+      // DOB
+      new Card({
+        source: {
+          label: "Automate Medical, Inc.",
+          url: "https://www.automatemedical.com/",
+        },
+        summary: `Date of birth: ${data.patient.birthDate}`,
+        indicator: "info",
+      }),
+      // Active
+      new Card({
+        detail: `${data.patient.active === true ? "Yes" : "No"}`,
```

Comment:
```suggestion
        detail: data.patient.active ? "Yes" : "No",
```

You don't need the template string literals (the `${}` stuff) because the return value of the expression is a string

You don't need to evaluate === true because in the ternary, `data.patient.active` is evaluated for truth**iness**: https://developer.mozilla.org/en-US/docs/Glossary/Truthy

Reply:
I think there are probably a bunch of places where you have this. Another example is:

```
encounters.length <= 1 ? "" : "s"
```

You can actually do `encounters.length ? "s" : ""`  (note I switched the return values)

The reason is because in JS >= 1 is truthy